kai/evaluation.py

```diff
@@ -150,23 +242,28 @@ def levenshtein_distance(s1, s2) -> float:
     return float(distances[-1])
 
 
-def compare_from_cli():
-    parser = argparse.ArgumentParser(description="Process a list of filepaths.")
-    parser.add_argument("filepaths", nargs="+", help="List of filepaths to process")
+# def compare_from_cli():
+#     parser = argparse.ArgumentParser(description="Process a list of filepaths.")
+#     parser.add_argument("filepaths", nargs="+", help="List of filepaths to process")
 
-    args = parser.parse_args()
+#     args = parser.parse_args()
 
-    configs: dict[str, BenchmarkConfiguration] = {}
-    for filepath in args.filepaths:
-        try:
-            with open(filepath, "r") as f:
-                configs[filepath] = KaiConfigModels.model_validate(yaml.safe_load(f))
+#     configs: dict[str, BenchmarkConfiguration] = {}
+#     for filepath in args.filepaths:
+#         try:
+#             with open(filepath, "r") as f:
+#                 configs[filepath] = KaiConfigModels.model_validate(yaml.safe_load(f))
 
-        except Exception as e:
-            print(f"Failed to load {filepath}: {e}")
+#         except Exception as e:
+#             print(f"Failed to load {filepath}: {e}")
 
-    print_nicely_formatted_comparison(evaluate(configs))
+#     print_nicely_formatted_comparison(evaluate(configs))
 
 
 if __name__ == "__main__":
-    compare_from_cli()
```

Comment:
The idea with this was that we could provide different config.tomls and have them run against the benchmarks, to make it super easy to compare two models or a new config without needing to get into the weeds

Reply:
Reimplemented!